src/mixins/highlightConfig.js

```diff
@@ -55,49 +58,42 @@ export default {
   methods: {
     setHighlight() {
       if (this.highlighted) {
-        this.shapeView.highlight(this.shapeBody, {highlighter: defaultHighlighter});
+        this.shapeView.highlight(this.shapeBody, { highlighter: defaultHighlighter });
         return;
       }
 
-      this.shapeView.unhighlight(this.shapeBody, {highlighter: defaultHighlighter});
+      this.shapeView.unhighlight(this.shapeBody, { highlighter: defaultHighlighter });
     },
     setErrorHighlight() {
       if (!this.shapeView) {
         return;
       }
-
-      if (this.hasError) {
+      if (this.hasError && this.autoValidate) {
         this.shapeView.highlight(null, { highlighter: errorHighlighter });
-      } else {
-        this.shapeView.unhighlight(null, { highlighter: errorHighlighter });
+        return;
       }
+      this.shapeView.unhighlight(null, { highlighter: errorHighlighter });
     },
-    configureCrown() {
-      this.paper.once('render:done', () => this.setHighlight());
-
-      const shapeView = this.shape.findView(this.paper);
+    setUpCrownConfig() {
```

Comment:
I guess this could be `setUpHighlightConfig` now. Instead of the method name it was originally under in the crownConfig.

Reply:
Good point!